main.py

```diff
@@ -1,16 +1,34 @@
-# This is a sample Python script.
+from flask import Flask, request, render_template, redirect, url_for, make_response
 
-# Press Shift+F10 to execute it or replace it with your code.
-# Press Double Shift to search everywhere for classes, files, tool windows, actions, and settings.
+app = Flask(__name__)
 
+@app.route('/')
+def index():
+    return render_template("start.html")
 
-def print_hi(name):
-    # Use a breakpoint in the code line below to debug your script.
-    print(f'Hi, {name}')  # Press Ctrl+F8 to toggle the breakpoint.
+@app.post('/start/')
```

Comment:
Адрес может быть тем же. Все равно он у вас не отрабатывает
```suggestion
@app.post('/')
```